### DISCUSSION TITLE
Retrieve Value from Set of Strings DataType

### DISCUSSION THREAD

**Discussion by ramyabitorder** [SalesforceCommerceCloud/sfcc-ci#413] [Team Posts]
Hello Team,

How do I retrieve value from a custom  field with data type 'set of strings'. Currently when execute productObj.activity = eachProduct.custom.activity ,   i get something like "Ljava.lang.String@746347". 

Thanks in Advance

**Comment by shubhamgoel710** [comment]
Hi,
While retrieving a value from a field with data type 'set of strings', it returns the Array Object itself.
To resolve this you can try the following:
**productObj.activity = eachProduct.custom.activity.slice();**
(Considering 'activity' to be your custom field with data type 'set of strings').

Thanks

**Comment by ramyabitorder** [comment]
Thank you so much Shubham, it worked
